pulp_rpm/tests/functional/api/test_publish.py

```diff
@@ -417,20 +423,24 @@ def get_checksum_types(self, **kwargs):
 
         return repomd_checksum_types, primary_checksum_types
 
-    def test_unspecified_checksum_types(self):
-        """Sync and publish an RPM repository and verify the checksum."""
-        repomd_checksum_types, primary_checksum_types = self.get_checksum_types()
+    def test_on_demand_unspecified_checksum_types(self):
+        """Sync and publish an RPM repository and verify the checksum types."""
+        repomd_checksum_types, primary_checksum_types = self.get_checksum_types(
```

Comment:
o_O, I didn't realize that `get_checksum_types` implies sync/publish. I know it was this way, just the name is a bit misleading and put me off for a moment :)